cmake/BareosCpmPackages.cmake

```diff
@@ -32,10 +32,45 @@ if(FETCHCONTENT_FULLY_DISCONNECTED AND NOT CPM_LOCAL_PACKAGES_ONLY)
 endif()
 include(CPM)
 
-CPMAddPackage(
+# Keep module alphabetically ordered
+
+CPMFindPackage(
+  NAME CLI11
+  VERSION 2.4.2
+  GITHUB_REPOSITORY CLIUtils/CLI11
+  EXCLUDE_FROM_ALL YES
+)
+
+CPMFindPackage(
+  NAME CPMLicenses.cmake 
+  GITHUB_REPOSITORY cpm-cmake/CPMLicenses.cmake
+  VERSION 0.0.5
+)
```

Comment:
While I like the idea of this, I'm not sure it won't work for us.
At least you will get different results depending on which packages were downloaded and which one were present locally.